.eslintrc

```diff
@@ -1,6 +1,230 @@
 {
+  "env": {
+    "browser": true,
+    "node": true,
+    "es6": true,
+    "mocha": true,
+  },
+  "globals": {
+    "nconf": true,
+    "assert": true,
+    "expect": true,
+    "fixture": true,
+  },
+  "parser": "babel-eslint",
+  "plugins": [],
+
   "parserOptions": {
     "ecmaVersion": 6,
-    "sourceType": "module"
+    "sourceType": "module",
+    "ecmaFeatures": {
+        "arrowFunctions": true,
+        "binaryLiterals": true,
+        "blockBindings": true,
+        "classes": true,
+        "defaultParams": true,
+        "destructuring": true,
+        "forOf": true,
+        "generators": true,
```

Comment:
Generators are not fully supported in babel-preset-es-2015. If you want to use generators we should update the babel ecma version.